typescript/src/schema/proto.ts

```diff
@@ -12,17 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/*eslint no-warning-comments: [0, { "terms": ["todo"], "location": "anywhere" }]*/
```

Comment:
Let's talk about this one folks @bcoe @alexander-fenster - do we want to broadly disable this rule?

Reply:
Yes, it does not make sense for us. And having TODOs in samples is totally fine.

(this specific TODO can be removed though :) )

Reply:
we have `TODO: disable pagination should be removed once next version of talent API is published` in this file, I think we need this eslint here?

Reply:
Oh, it's _that_ TODO. Yes, we need it.

Reply:
We yanked the upstream rule, should be able to remove this

Reply:
removed /*eslint no-warning-comments: .../ line since we disable this check globally in `gts`